Dockerfile

```diff
@@ -40,12 +40,12 @@ RUN wget --directory-prefix=${TEMP_DIR} ${OC_DL_URL} && \
 ENV PATH=${PATH}:"/usr/local/go/bin":${GOPATH}/"bin"
 
 # Git identifier to checkout
-ARG TNF_VERSION
-ARG TNF_SRC_URL=https://github.com/test-network-function/test-network-function
+ARG TNF_VERSION=remove-partner-pod
```

Comment:
I guess these changes will be reverted before merging, right? ;)

Reply:
right, this is used to allow .github CI to pass without issues.